content/toolsAndResources/nge/ngeBlocks.md

```diff
@@ -1946,6 +2022,44 @@ There are times when it is necessary to recompute normals when creating geometry
 #### Outputs
 - **output** is the geometry after computing normals.
 
+<H3Image title="Lattice" image="/img/tools/nge/lattice.jpg" alt="Lattice Node"/>
+The lattice places a simplified "cage" over your geometry, allowing you to manipulate the geometry from a higher more simplified level. This is similar in concept to how subdivisional surfaces work, where a complex mesh is derived from a more simplified mesh. The lattice tool is a handy way to make larger manipulations to an already detailed piece of geometry.

```

Comment:
"higher more simplified level" -> "higher and simplified level." I would also say that a lattice is more of a deformer based on a simplified mesh more than comparing it to a subdivision surface. In a subdivision, the mesh may have less points than the final, but the simplified mesh has the general silhouette of the final subdivision surface. The lattice is a cube with a specified resolution that can deform a mesh that bears no similarity to the silhouette of a cube. It's a little confusing to me to bring in subdivisions when speaking about lattice deformers.